src/actions/plugins/easymotion/easymotion.ts

```diff
@@ -81,9 +76,12 @@ export class EasyMotion {
    */
   public clearDecorations() {
     const editor = vscode.window.activeTextEditor!;
+
     for (let i = 1; i <= this.decorations.length; i++) {
       editor.setDecorations(EasyMotion.getDecorationType(i), []);
     }
+
+    this.fade?.dispose();
```

Comment:
Why create and dispose of this each time, rather than just `setDecorations(fade, [])`?

Reply:
i believe that either way is the same because `setDecorations(fade, [])` calls disposes anyway